src/blog/templates/blog/includes/brazil-time.html

```diff
@@ -0,0 +1,21 @@
+{% load brazil_time %}
+
+<span id="brazilTime" class="p-1 rounded text-xs bg-gray-700 text-gray-100">
```

Comment:
I think it will be prettier if the background matches the blue, so bg-blue-800

Reply:
```suggestion
<span id="brazilTime" class="p-1 rounded text-xs bg-blue-800 text-gray-100">
```